.github/workflows/build.yaml

```
@@ -61,11 +104,12 @@ jobs:
   # Package & Upload to S3 on main or manual dispatch
   # ────────────────────────────────────────────────────────────────
   package:
-    needs: build-test
+    needs: [prepare, build-test]
     if: >
```

actionlint (1.7.7)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)